website/src/contribute/release-guide.md

```diff
@@ -595,6 +600,9 @@ For this step, we recommend you using automation script to create a RC, but you
   1. Update last release download links in `website/src/get-started/downloads.md`.
   1. Update `website/src/.htaccess` to redirect to the new version.
   1. Build and stage python wheels.
+  1. Public staging artifacts
```

Comment:
I also had issues with figuring this out, but it is written already in line 588 above. Not sure if it's worth repeating so close by.

Reply:
Would it be worth removing from line 588?

Reply:
Probably, yes, since we both skimmed over that list above and missed the comment, but I do remember looking at this one. I'd move this to be the first item (chronological order).